app/src/main/java/io/apicurio/registry/services/http/RegistryExceptionMapperService.java

```diff
@@ -129,6 +128,13 @@
         map.put(TenantManagerClientException.class, HTTP_INTERNAL_ERROR);
         map.put(ParametersConflictException.class, HTTP_CONFLICT);
         map.put(DownloadNotFoundException.class, HTTP_NOT_FOUND);
+        // From io.apicurio.registry.mt.TenantMetadataService:
+        map.put(NotAuthorizedException.class, HTTP_FORBIDDEN);
+        map.put(ForbiddenException.class, HTTP_FORBIDDEN);
+        map.put(RegistryTenantNotFoundException.class, HTTP_NOT_FOUND);
```

Comment:
I wonder if we should instead map this to HTTP_FORBIDDEN.  I think returning a 404 if the registry tenant is not found has the side effect of leaking information.  This can be used by an attacker to scan for valid tenant IDs.

WDYT?

Reply:
Does the PR from @carlesarnal impact this?  I suspect so.

Reply:
Yes, it does.